packages/account/src/Components/poi/idv-document-submit/__tests__/idv-document-submit.spec.tsx

```diff
@@ -126,25 +134,27 @@ describe('<IdvDocumentSubmit/>', () => {
         expect(document_number_input.name).toBe('document_number');
         expect(document_number_input).toBeDisabled();
 
-        fireEvent.change(document_type_input, { target: { value: 'Test document 2 name' } });
-        expect(document_number_input).toBeEnabled();
+        userEvent.selectOptions(document_type_input, 'Test document 2 name');
+        await waitFor(() => {
+            expect(document_number_input).toBeEnabled();
+        });
         expect(screen.queryByText(/please enter the correct format/i)).not.toBeInTheDocument();
-
+        (isDocumentNumberValid as jest.Mock).mockReturnValueOnce('please enter your document number');
         fireEvent.blur(document_number_input);
```

Comment:
can we use `userEvent.unhover`

Reply:
Testcase fails when userEvent.unhover is used

Reply:
yes, that’s why I replied about it. in some reason line 142 is not working with userEvent. so there are two lines left with fireEvent